fix: update incorrect edgexfoundry-holding imports

siggiskulason opened the pull request:
Import statements in Go code were still using github.com/edgexfoundry-holding, rather than
github.com/edgexfoundry. Even though the edgexfoundry-holding repository is redirected to edgexfoundry on github.com,
this means that the package namespace is different. Therefore, setting the version in the Makefile did not work correctly and
the version returned by http://localhost:49985/api/version was incorrect.

This PR fixes #60

Signed-off-by: Siggi Skulason <siggi.skulason@canonical.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [NA] Tests for the changes have been added (for bug fixes / features)
- [NA] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior and link to a relevant issue. -->
curl -X GET http://localhost:49985/api/version returned "to be replaced by makefile"
Issue Number: #60 


## What is the new behavior?
curl -X GET http://localhost:49985/api/version returns the correct version string

## Does this PR introduce a breaking change?
<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->

- [ ] Yes
- [x] No

## Are there any new imports or modules? If so, what are they used for and why?
This bugfix fixes incorrect imports - replacing "github.com/edgexfoundry-holding/device-camera-go" with "github.com/edgexfoundry/device-camera-go"

## Are there any specific instructions or things that should be known prior to reviewing?

## Other information

tonyespy reviewed (changes requested):
Changes look good to me. Again, the only thing I'd tidy up is the git commit message. One other rule of thumb is to try and keep your git commit summary <= 50 characters ([How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)).

In this case you could again simplify to something like:

`fix: update incorrect edgexfoundry-holding imports
`

siggiskulason commented:
@tonyespy Thanks - that article on git messages was interesting as well. I've updated the commit message and changed the title of this PR accordingly as well. Could you merge this if you are ok with the changes? - Siggi

lenny-intel reviewed (approved):
LGTM
